### PR TITLE
[VMware Solution]Api2021-06-01 release

### PR DESCRIPTION
**Pull request by lijinpei2008**
## Description

AVS cmdlet test for powershell.

### CODE REVIEW

**Review by VeryEarly — CHANGES_REQUESTED** [review]
please mask subscription ids in examples

File: src/VMware/docs/Get-AzVMwareGlobalReachConnection.md [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+Name          Type
+----          ----
+azps_test_grc Microsoft.AVS/privateClouds/globalReachConnections
+```
+
+Get a global reach connection by name in a private cloud
+
+### Example 3: Get a global reach connection by InputObject in a private cloud
+```powershell
+PS C:\>  Get-AzVMwareGlobalReachConnection -InputObject "/subscriptions/ba75e79b-dd95-4025-9dbf-3a7ae8dff2b5/resourceGroups/azps_test_group/providers/Microsoft.AVS/privateClouds/azps_test_cloud/globalReachConnections/azps_test_grc"
```
Comment (VeryEarly) [inline src/VMware/docs/Get-AzVMwareGlobalReachConnection.md:62]: this looks like resource id not input object

**Review by VeryEarly — CHANGES_REQUESTED** [review]
please double check "-InputObject" parameters, they look like resource id

File: src/VMware/docs/Get-AzVMwarePrivateCloudAdminCredential.md [hunk 2]
```diff
@@ -22,16 +22,27 @@ List the admin credentials for the private cloud
 
 ## EXAMPLES
 
-### Example 1: Get admin credential of private cloud
+### Example 1: List the admin credentials for the private cloud
```
Comment (VeryEarly) [inline src/VMware/docs/Get-AzVMwarePrivateCloudAdminCredential.md:25]: Will there be multiple credentials for one private cloud?

File: src/VMware/docs/New-AzVMwareAddonSrmPropertiesObject.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+# New-AzVMwareAddonSrmPropertiesObject
+
+## SYNOPSIS
+Create a in-memory object for AddonSrmProperties
+
+## SYNTAX
+
+```
+New-AzVMwareAddonSrmPropertiesObject -LicenseKey <String> [<CommonParameters>]
```
Comment (VeryEarly) [inline src/VMware/docs/New-AzVMwareAddonSrmPropertiesObject.md:16]: why is this needed when there is only one property?

File: src/VMware/docs/New-AzVMwareAuthorization.md [hunk 4]
```diff
@@ -23,18 +23,16 @@ Create or update an ExpressRoute Circuit Authorization in a private cloud
 
 ## EXAMPLES
 
-### Example 1: Create autorization
+### Example 1: Create an  Circuit Authorization in a private cloud
```
Comment (VeryEarly) [inline src/VMware/docs/New-AzVMwareAuthorization.md:26]: please remove the extra white spaces

**Review by VeryEarly — CHANGES_REQUESTED** [review]
for in-memory objects, please provide examples how they used by other cmdlets

File: src/VMware/utils/Unprotect-SecureString.ps1 [hunk 5]
```diff
@@ -0,0 +1,16 @@
+#This script converts securestring to plaintext
+
+param(
+    [Parameter(Mandatory, ValueFromPipeline)]
+    [System.Security.SecureString]
+    ${SecureString}
+)
+
+$ssPtr = [System.Runtime.InteropServices.Marshal]::SecureStringToBSTR($SecureString)
```
Comment (VeryEarly) [inline src/VMware/utils/Unprotect-SecureString.ps1:9]: this line also need to be in try finally block

File: src/VMware/test/Remove-AzVMwarePrivateCloud.Tests.ps1 [hunk 6]
```diff
@@ -12,11 +12,16 @@ while(-not $mockingPath) {
 . ($mockingPath | Select-Object -First 1).FullName
 
 Describe 'Remove-AzVMwarePrivateCloud' {
-    It 'Delete' -skip {
-        { throw [System.NotImplementedException] } | Should -Not -Throw
+    It 'Delete' {
```
Comment (VeryEarly) [inline src/VMware/test/Remove-AzVMwarePrivateCloud.Tests.ps1:15]: where was this private cloud been created?

Comment (lijinpei2008) [reply src/VMware/test/Remove-AzVMwarePrivateCloud.Tests.ps1:15]: We need to create two private Clouds to test the delete command separately.
One created in Get-AzVMwarePrivateCloud.
One created in New-AzVMwarePrivateCloud.

File: src/VMware/docs/Az.VMware.md [hunk 7]
```diff
@@ -36,10 +36,10 @@ List the admin credentials for the private cloud
 Create or update a addon in a private cloud
 
 ### [New-AzVMwareAddonSrmPropertiesObject](New-AzVMwareAddonSrmPropertiesObject.md)
-Create a in-memory object for AddonSrmProperties
+Create a Srm object for AddonSrmProperties
```
Comment (VeryEarly) [inline src/VMware/docs/Az.VMware.md:39]: an Srm

File: src/VMware/docs/New-AzVMwareAddonSrmPropertiesObject.md [hunk 8]
```diff
@@ -8,7 +8,7 @@ schema: 2.0.0
 # New-AzVMwareAddonSrmPropertiesObject
 
 ## SYNOPSIS
-Create a in-memory object for AddonSrmProperties
+Create a Srm object for Addon
```
Comment (VeryEarly) [inline src/VMware/docs/New-AzVMwareAddonSrmPropertiesObject.md:11]: an Srm

File: src/VMware/docs/New-AzVMwareAddonSrmPropertiesObject.md [hunk 9]
```diff
@@ -17,11 +17,11 @@ New-AzVMwareAddonSrmPropertiesObject -LicenseKey <String> [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-Create a in-memory object for AddonSrmProperties
+Create a Srm object for Addon
```
Comment (VeryEarly) [inline src/VMware/docs/New-AzVMwareAddonSrmPropertiesObject.md:20]: an Srm

File: src/VMware/docs/New-AzVMwareAddonSrmPropertiesObject.md [hunk 10]
```diff
@@ -17,11 +17,11 @@ New-AzVMwareAddonSrmPropertiesObject -LicenseKey <String> [<CommonParameters>]
...
 
 ## EXAMPLES
 
-### Example 1: Create a in-memory object for AddonSrmProperties
+### Example 1: Create a Srm object for Addon
```
Comment (VeryEarly) [inline src/VMware/docs/New-AzVMwareAddonSrmPropertiesObject.md:24]: an Srm

File: src/VMware/docs/New-AzVMwareAddonSrmPropertiesObject.md [hunk 11]
```diff
@@ -30,7 +30,7 @@ AddonType ProvisioningState LicenseKey
 SRM                         YourLicenseKeyValue
 ```
 
-Create a in-memory object for AddonSrmProperties
+Create a Srm object for Addon
```
Comment (VeryEarly) [inline src/VMware/docs/New-AzVMwareAddonSrmPropertiesObject.md:33]: an Srm

File: src/VMware/test/Remove-AzVMwarePrivateCloud.Tests.ps1 (hunk 6 above)
Comment (VeryEarly) [reply src/VMware/test/Remove-AzVMwarePrivateCloud.Tests.ps1:15]: we do not want to create and delete resources in separate test cases, we do not promise the order of test cases executed.

what it this test case executed before get and new?

File: src/VMware/docs/Get-AzVMwarePrivateCloudAdminCredential.md (hunk 2 above)
Comment (VeryEarly) [reply src/VMware/docs/Get-AzVMwarePrivateCloudAdminCredential.md]: before you revert your change, can you please confirm whether there be multiple credentials for one private cloud?

**Review by VeryEarly — APPROVED** [review]
